patsicko/my-brand-bakend
# Ft blog retrieve all 006

## What this PR do
Retrieve all blogs
## Description of the tasks to be completed
- GET /api/allBlogs
[Finishes#006]

## How should this be manually tested
- open postman
- set your HTTP request to GET
- input API endpoint
- Click Send.
- Status: 200 OK should be displayed
- Retrieved data are showing up in the body enveloped in JSend object

## Trello story id
#006
